plugins/destination/s3/docs/_authentication.md

```diff
@@ -1,85 +1,37 @@
-The plugin needs to be authenticated with your account(s) in order to sync information from your cloud setup.
+---
+hub-title: Authentication
```

Comment:
```suggestion
```

`Authentication` is already the default title for a file named `authentication.md`